Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Block_IP.yml

```diff
@@ -0,0 +1,733 @@
+    nexttasks:
+      '#default#':
+      - "5"
+      "yes":
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: inputs.StaticAddressGroupName
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 170,
+          "y": 410
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 99f457e4-1a30-4314-8bf7-7bb4f9b6a8e6
+    type: title
+    task:
+      id: 99f457e4-1a30-4314-8bf7-7bb4f9b6a8e6
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1680
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 4b2a7ae5-8d45-4e88-86b1-fa514d8d1d07
+    type: regular
+    task:
+      id: 4b2a7ae5-8d45-4e88-86b1-fa514d8d1d07
+      version: -1
+      name: Get the Address Group List
+      description: Lists all address groups.
+      script: '|||prisma-sase-address-group-list'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      folder:
+        complex:
+          root: inputs.Folder
+      group_id:
+        complex:
+          root: inputs.StaticAddressGroupName
+      name:
+        complex:
+          root: inputs.StaticAddressGroupName
+      tsg_id:
+        complex:
+          root: inputs.TSGID
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -260,
+          "y": 580
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 029e1e65-93e4-45e9-82bd-22e586cc5b5e
+    type: regular
+    task:
+      id: 029e1e65-93e4-45e9-82bd-22e586cc5b5e
+      version: -1
+      name: Create a new Static Address Group
+      description: Create a new address group.
+      script: '|||prisma-sase-address-group-create'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "6"
+    scriptarguments:
+      folder:
+        complex:
+          root: inputs.Folder
+      name:
+        simple: Block IP group - Created by XSOAR
+      static_addresses:
+        complex:
+          root: PrismaSase.Address
+          accessor: name
```

Comment:
Is the PrismaSase.Address.name eq the IP address?

Reply:
The reason was that the command needs to have the object name - that's why I'm creating the objects 2 tasks before it.